contracts/misc/ProfileCreationProxy.sol

```diff
@@ -0,0 +1,44 @@
+// SPDX-License-Identifier: AGPL-3.0-only
```

Comment:
We changed license to MIT. I'll be making a PR updating the license on every contract, would be nice to change it here too so it's merged up to date.
```suggestion
// SPDX-License-Identifier: MIT
```